docs/rules/semi.md

```diff
@@ -133,6 +141,54 @@ var name = "ESLint"
 })()
 ```
 
+#### "always-except-oneline"
```

Comment:
I think this could be confusing for people who may conflate "except" with "accept".  It reads too much like a sentence.

This actually seems like it should be a separate option, like:

```json
{
    "semi": [2, "always", "except-oneline"]
}
```

Not sure about `oneline` vs. `single-line`, but I think I like `oneline` better.